org.eclipse.bluesky/build.properties

```diff
@@ -3,5 +3,5 @@ output.. = bin/
 bin.includes = META-INF/,\
                .,\
                plugin.xml,\
-               grammars/,\
-               language-servers/
+               language-servers/,\
+               language-clients/
```

Comment:
Can we call it `language-configurations` to better map to the goal and content of this folder?

Reply:
done